homeassistant/components/ezviz/camera.py

```diff
@@ -198,7 +198,7 @@ async def async_setup_entry(
     )
 
     platform.async_register_entity_service(
-        SERVICE_ALARM_TRIGER,
+        SERVICE_ALARM_TRIGGER,
```

Comment:
This changes code, have you been able to test this using an ezviz camera to make sure it still works as expected?